client/scss/components/_blockquote.scss

```diff
@@ -0,0 +1,56 @@
+//* @define blockquote; weak
+
+.blockquote {
+  @include font('HNM2');
+  text-align: center;
+
+  &.blockquote--wb {
```

Comment:
It's to enforce needing both on the element (if it's to be modified, obviously).